src/VirtoCommerce.XCatalog.Core/Schemas/PropertyType.cs

```diff
@@ -67,6 +71,22 @@ public PropertyType(IMediator mediator, IDataLoaderContextAccessor dataLoader)
             Field<StringGraphType>("valueId")
                 .Resolve(context => context.Source.Values.Select(x => x.ValueId).FirstOrDefault());
 
+            var group = new FieldType
+            {
+                Name = "group",
+                Type = GraphTypeExtensionHelper.GetActualType<PropertyGroupType>(),
+                Resolver = new FuncFieldResolver<Property, IDataLoaderResult<PropertyGroup>>(context =>
+                {
+                    var loader = dataLoader.Context.GetOrAddBatchLoader<string, PropertyGroup>("propertyGroupLoader", async (ids) =>
+                    {
+                        var groups = await propertyGroupService.GetNoCloneAsync(ids.ToList());
+                        return groups.ToDictionary(x => x.Id);
+                    });
+                    return loader.LoadAsync(context.Source.PropertyGroupId);
+                })
+            };
+            AddField(group);
```

Comment:
There will be a lot of duplicate data in the response if many properties belong to the same property group

Reply:
@artem-dudarev 
There won't be duplication even if all properties have the same property group - Dataloader takes care of data duplication problem